release_bp/vendor/dell/opx_core_stretch.xml

```diff
@@ -12,6 +12,6 @@
   <xi:include href="../../repo/stretch_updates.xml"/>
 
   <package_list>
-    <package name="linux-image-4.9.0-8-amd64" version="[4.9.110-3+deb9u4opx3]"/>
+    <package name="linux-image-4.9.110" version="[4.9.110-3+deb9u4-99999]"/>
```

Comment:
The last part of the kernel version number (99999) should be a build generation ID incremented by its kernel build (that is, should be a monotonically increasing number provided by the package build/release system).  The number 99999 was specifically used to mark a developer built package.

Reply:
There is a change required to the build environment, that is -- the build environment maintains the current version (build generation ID, local version number ... ), and sets BUILD_NUMBER to the desired number for this part of the version number (there is a "standard" way to provide this to the kernel build script)


There should be a concept of BUILD_NUMBER in the OPX builds.  When we build in open-flow this is all handled for all production builds.  I expect there is some method to track the build number of repositories/packages.


I'll approve the pull request, but I believe something that should just work, doesn't.  (the build_number I expect to be part of the build environment, and exported so the make variable (macro -- take your choice) is set correctly during the build and exported so make will pick it up.

Reply:
@famleebob could you provide the approval as mentioned? Fixing the process should also be done in OS10.